Documentation/gettingstarted/k8s-install-restart-pods.rst

```diff
@@ -20,3 +20,10 @@ connectivity provided by Cilium and NetworkPolicy applies to them:
     pod "kube-state-metrics-7d9774bbd5-n6m5k" deleted
     pod "l7-default-backend-6f8697844f-d2rq2" deleted
     pod "metrics-server-v0.3.1-54699c9cc8-7l5w2" deleted
+
+.. note::
+
+    This may error out on OSX due to ``-r`` being unsupported by
```

Comment:
I think the "new" branding is macOS rather than OSX:
```suggestion
    This may error out on macOS due to ``-r`` being unsupported by
```

Reply:
Applied, thanks!